aggoracle/chaingersender/evm.go

```diff
@@ -99,32 +166,87 @@ func (c *EVMChainGERSender) IsGERInjected(ger common.Hash) (bool, error) {
 	return gerIndex.Cmp(common.Big0) == 1, nil
 }
 
+// IsGERProposed checks if the provided global exit root has already been proposed by the oracle committee member
+func (c *EVMChainGERSender) IsGERProposed(ger common.Hash) (bool, error) {
+	isInjected, err := c.IsGERInjected(ger)
+	if err != nil {
+		return false, fmt.Errorf("failed to check if global exit root is injected %s: %w", ger, err)
```

Comment:
It doesn't make sense to provide the same error message wrapped twice, so probably just propagate the retrieved error instead. WDYT?

```suggestion
		return false, err
```

Reply:
done